overlay/osx/qt5-base/portfile.cmake

```diff
@@ -9,6 +9,7 @@ include(install_qt)
 
 #########################
 ## Find Host and Target mkspec name for configure
+
```

Comment:
stray new line

Reply:
I kept that newline since the formatting looked more consistent to me that way, but I'd be fine with removing it again too.